.travis.yml

```diff
@@ -26,15 +26,10 @@ matrix:
     - php: 7.2
       env: VARNISH_VERSION=4.1 VARNISH_MODULES_VERSION=0.9.1
 
-      # Test Symfony LTS versions
-    - php: 7.2
-      env: DEPENDENCIES="symfony/lts:^2"
-    - php: 7.2
-      env: DEPENDENCIES="symfony/lts:^3 toflar/psr6-symfony-http-cache-store:^1.0"
```

Comment:
As we've bumped to 3.4 as a minimum requirement I thought `--prefer-lowest` would already cover that as "lowest" === "3.4" now. Wrong?

Reply:
i'd keep both so that we know if its only the oldest symfony 3.4 that fails or also the newest 3.4. (and also the other way round, we did have some regressions in the past, where some change in a patch version of old symfony broke stuff)